package.json

```diff
@@ -8,7 +8,9 @@
     "test:watch": "jest --watch",
     "build": "webpack -p ",
     "start:dev": "webpack -d && webpack-dev-server --mode development --open --hot",
-    "lint": "./node_modules/.bin/eslint"
+    "lint": "./node_modules/.bin/eslint",
+    "start": "node server.js",
+    "heroku-postbuild": "webpack -p"
```

Comment:
@Philipotieno this command is redundant compare with build command on line 9. so it means heroku runs `webpack -p` twice